functests/0_config/test_suite_performance_config_test.go

```diff
@@ -0,0 +1,24 @@
+// +build !unittests
+
+package __performance_config_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"testing"
+
+	ginkgo_reporters "kubevirt.io/qe-tools/pkg/ginkgo-reporters"
+
+	"github.com/openshift-kni/performance-addon-operators/functests/utils/junit"
```

Comment:
You did not import the ` __performance_config` here, so it won't be executed.

Reply:
Seems it is not needed, everything is executed as expected. I guess `*_test` packages import their `*` package implicitly?

Reply:
Ah yes, that's it. I did not notice they are in the same folder.